mods/parsoid.js

```diff
@@ -615,10 +611,18 @@ PSP.makeTransform = function(from, to) {
         return transform
         .then(function(res) {
             // Unwrap to the flat response format
-            var innerRes = res.body[to];
+            var innerRes = {};
+            if (to === 'wikitext') {
```

Comment:
It should probably be `if (to !== 'pagebundle')` instead.  The `html` and `wikitext` formats return the results in the main body of the response (as does RESTBase).  It's only the `pagebundle` format that returns a JSON object wrapping the content instead.  Again, help refactoring would be appreciated if you want to turn the default case around and specially-handle only the pagebundle case.